crates/bevy_text/src/lib.rs

```diff
@@ -76,7 +76,11 @@ use bevy_sprite::SpriteSystem;
 /// When the `bevy_text` feature is enabled with the `bevy` crate, this
 /// plugin is included by default in the `DefaultPlugins`.
 #[derive(Default)]
-pub struct TextPlugin;
+pub struct TextPlugin {
+    /// If [false], some characters (especially Unicode emojies) might not load properly due to unsupported font
```

Comment:
These docs need to be improved. More details on behavior, links to cosmic-text's information about this.